data-esnext.js

```diff
@@ -305,6 +308,7 @@ exports.tests = [
         ie11: false,
         firefox2: false,
         chrome77: false,
+        safari17: true,
```

Comment:
Note that the current `Set` methods test case is obsolete and largely incorrect. Like most other ESNext and latest ES versions tests. It's not directly related to this PR.

Reply:
sound like an issue or a PR should be filed to note/fix that, otherwise how could anybody know about it

Reply:
I don't know what about it is obsolete or incorrect. If you want to see it fixed, please file an issue with details.

Reply:
You know these methods, how their specification and implementations have changed, well enough, at least because you wrote polyfills for them and took part in agreeing on the specification. Those tests are very simple and you just need to look at the test code. So I see only 2 options - you are either too lazy to look at the code, or you are lying.

After you started to block my improvements of this repo (and almost killed this project), I don't care if will it be fixed or not - for my cases, I have another compatibility data project. I just reported it as a comment since it could be interesting for someone.

Reply:
If you didn't care, you wouldn't review these PRs or make these comments.

Reply:
Please don't tell me what to do and what not to do - and I won't tell you where to go -)

Reply:
I didn't. I told you what your actions and behavior reveal, which is that you do, indeed, care very much.

Reply:
Because of your actions and behavior, this project hasn't actual tests and results for modern ES versions and actual proposals and is almost dead - and after that, you tell me something about my actions and behavior -)

Reply:
https://github.com/nodejs/TSC/issues/1445#issuecomment-1745405198

Reply:
https://yourlogicalfallacyis.com/tu-quoque

Reply:
No, darling, wrong card. Think again, it's not difficult.